internal/server/proto/server.proto

```diff
@@ -1979,6 +1998,12 @@ message ExecStreamRequest {
 
     // Pty is set if we should allocate a PTY for this exec stream.
     PTY pty = 3;
+
+    // Instance to send the exec request to. This is indicates that
+    // the client wants an exec session to this specific instance,
+    // rather than one the server picks. Targetted instances can be
+    // any instance type, where as untargeted have to be LONG_RUNNING.
+    string target_instance_id = 4;
```

Comment:
Given `deployment_id` doesn't have a `target_` prefix, I'm gonna bike shed and say this one shouldn't either.

Also, unclear: if this is specified, should I _not_ specify deployment ID?

We may want to change it to this:

```proto
oneof target {
  string deployment_id = 1;
  string instance_id = 4;
}
```

(As long as you keep the same tag numbers, that is backwards compat)

Reply:
Totes fair. Also good call on the use of oneof here, that's the right move.